pephubclient/helpers.py

```diff
@@ -47,20 +47,54 @@ def send_request(
         )
 
     @staticmethod
-    def decode_response(response: requests.Response, encoding: str = "utf-8") -> str:
+    def decode_response(
+        response: requests.Response, encoding: str = "utf-8", output_json: bool = False
+    ) -> Union[str, dict]:
         """
         Decode the response from GitHub and pack the returned data into appropriate model.
 
         :param response: Response from GitHub.
         :param encoding: Response encoding [Default: utf-8]
+        :param output_json: If True, return response in json format
         :return: Response data as an instance of correct model.
         """
 
         try:
-            return response.content.decode(encoding)
+            if output_json:
+                return response.json()
+            else:
+                return response.content.decode(encoding)
         except json.JSONDecodeError as err:
             raise ResponseError(f"Error in response encoding format: {err}")
 
+    @staticmethod
+    def parse_query_param(pep_variables: dict) -> str:
+        """
+        Grab all the variables passed by user (if any) and parse them to match the format specified
+        by PEPhub API for query parameters.
+
+        :param pep_variables: dict of query parameters
+        :return: PEPHubClient variables transformed into string in correct format.
+        """
+        parsed_variables = []
+
+        for variable_name, variable_value in pep_variables.items():
+            parsed_variables.append(f"{variable_name}={variable_value}")
+        return "?" + "&".join(parsed_variables)
```

Comment:
How about using `urllib.parse.urlencode` for this? This looks like it works, but `urllib` is [vanilla python](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode).

Would look like this:
```python
from urllib.parse import urlencode

params = {
    'apple': 5,
    'orange': 4
}
query_string = urlencode(params)
print(query_string) # 'apple=5&orange=4'
```